understanding/reflow.html

```diff
@@ -86,9 +87,9 @@ <h3>コンテンツの可視性と可用性</h3>
                
                <h3>リフローにおけるコンテンツの例外</h3>
                
-               <p>利用や意味の理解のために 2 次元レイアウトを必須とするコンテンツは、意味を失うことなくリフローすることはできない。したがって、この達成基準の範囲外となる。例えば、画像及び動画は性質上 2 次元である。画像を切り取って縦に並べるとコンテンツは使用できなくなるため、範囲外となる。ただし、ズームレベルが 400 ％になっても、これらの要素をビューポートの境界内に収めたままにすることはできる (参考達成方法を参照)。</p>
+               <p>利用や意味の理解のために 2 次元レイアウトを必須とするコンテンツは、意味を失うことなくリフローすることはできない。したがって、この達成基準の範囲外となる。例えば、画像及び動画は性質上 2 次元である。画像を切り取って縦に並べるとコンテンツは使用できなくなるため、範囲外となる。このようなコンテンツの部分については、2 次元スクロールを提供することは許容される。ただし、ズームレベルが 400 ％になっても、これらの要素をビューポートの境界内に収めたままにすることはできる (参考達成方法を参照)。</p>
                
-               <p>複雑なデータテーブルは、見出しとデータセルの間に 2 次元の関係がある。この関係性はコンテンツの意味を伝えるために不可欠である。したがって、この達成基準はデータテーブル全体には適用されない。ただし、データテーブル内のセルは、用法又は意味のために 2 次元レイアウトを必要とする種類のコンテンツを含まない限り、サイズ要件内に収まるべきである。</p>
+               <p>複雑なデータテーブルは、見出しとデータセルの間に 2 次元の関係がある。この関係性はコンテンツの意味を伝えるために不可欠である。したがって、この達成基準はデータテーブル全体には適用されず、それらに対して 2 次元スクロールを提供することは許容される。ただし、データテーブル内のセルは、用法又は意味のために 2 次元レイアウトを必要とする種類のコンテンツを含まない限り、サイズ要件内に収まるべきである。</p>
```

Comment:
ここのthemはデータテーブルですから、こうでしょうか

```suggestion
               <p>複雑なデータテーブルは、見出しとデータセルの間に 2 次元の関係がある。この関係性はコンテンツの意味を伝えるために不可欠である。したがって、この達成基準はデータテーブル全体には適用されず、データテーブルに対して 2 次元スクロールを提供することは許容される。ただし、データテーブル内のセルは、用法又は意味のために 2 次元レイアウトを必要とする種類のコンテンツを含まない限り、サイズ要件内に収まるべきである。</p>
```

Reply:
@momdo さん、レビューありがとうございます。おっしゃるように、them は「データテーブル」のことです。改善案の通り「データテーブルに対して」と書いたほうがすんなりと頭に入ってきそうですね。